tools/ai/start_debugpy_e2e_check.py

```
@@ -0,0 +1,37 @@
+SLEEP_SECONDS = 12
+
+def main():
+    try:
+        import debugpy  # type: ignore
+    except Exception as e:
+        print("debugpy is not installed in Blender's Python environment:", e)
+        return 1
+
+    try:
+        print(f"E2E: Starting debugpy on port {PORT} (will sleep {SLEEP_SECONDS}s)")
+        debugpy.listen(("0.0.0.0", PORT))
```

Ruff (0.14.1)

Trace found: `debugpy.listen` used

(T100)